src/main/java/edu/kis/powp/jobs2d/drivers/SelectLineMenuOptionListener.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.shape.LineFactory;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectLineMenuOptionListener implements ActionListener {
+    private LineManager lineManager;
+
+    public SelectLineMenuOptionListener(LineManager lineManager) {
+        this.lineManager = lineManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        switch (e.getActionCommand()) {
+            case "Line dot": {
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.